webportal/webapp/src/environments/environment.dev-private.ts

```diff
@@ -1,15 +1,15 @@
 export const environment = {
   production: false,
-  STREAMING_URL: 'https://dev-proxy-portal.securedatacommons.com/guacamole/?authToken=',
+  STREAMING_URL: 'https://dev-portal.securedatacommons.com/guacamole/?authToken=',
   REGION: 'us-east-1',
   USER_POOL_ID: 'us-east-1_Y5JI7ysvY',
   CLIENT_ID: '22gr1lfintehcaac2cdmmsur52',
   IDENTITY_PROVIDER: 'dev-dot-ad',
   APP_DOMAIN: 'dev-sdc-dot-webportal',
-  REDIRECT_URL: 'https://dev-proxy-portal.securedatacommons.com/index.html', //dev environment
-  API_ENDPOINT: 'https://dev-proxy-portal.securedatacommons.com/webportal-api/',
+  REDIRECT_URL: 'https://dev-portal.securedatacommons.com/index.html',
+  API_ENDPOINT: 'https://dev-portal.securedatacommons.com/webportal-api/',
   LOGIN_GOV_COGNITO_APP_CLIENT_ID: '3ikuiqen3fdfl5brs5uk49cvsu',
-  ACCOUNT_LINK_URL: 'https://dev-proxy-portal.securedatacommons.com/account-link-dev',
+  ACCOUNT_LINK_URL: 'https://dev-portal.securedatacommons.com/account-link-dev',
```

Comment:
One small thing (I was about to add it in my branch) can you add:
RESET_TEMPORARY_PASSWORD_PATH: 'dev-reset-temporary-password',

Reply:
Done